src/app/components/common/node-editor/editor.ts

```diff
@@ -335,25 +335,29 @@ export function getUpdatedFilterPipeline() {
 
   for (let i = 0; i < nodes.length; i++) {
     if (i == 0) {
+      // check if starting node of pipeline i.e. south/storage is connected
       if (!connections.find(c => c.source === nodes[i].id)) {
         console.log("Dangling connection");
```

Comment:
not in the scope of this, but should we log more details in logs? Dangling connection for or from .. to ...